dtrace-parser/src/dtrace.pest

```diff
@@ -17,8 +17,11 @@ IDENTIFIER = @{ ASCII_ALPHA+ ~ (ASCII_ALPHANUMERIC | "_")* }
 BIT_WIDTH = @{ "8" | "16" | "32" | "64" }
 SIGNED_INT = ${ "int" ~ BIT_WIDTH ~ "_t" }
 UNSIGNED_INT = ${ "uint" ~ BIT_WIDTH ~ "_t" }
-STRING = { "char" ~ "*" }
-DATA_TYPE = { STRING | UNSIGNED_INT | SIGNED_INT}
+INT = $ { SIGNED_INT | UNSIGNED_INT }
+STAR = ${ "*" }
+INTEGER = ${ (INT ~ STAR) | INT }
```

Comment:
This is super nitpicky, but having both `INT` and `INTEGER` (that aren't synonymous) seems a little dubious. Names are hard; maybe something like `INTEGER` (for what is currently `INT`) and `INTEGER_TYPE` (for int or int pointer)? I don't feel strongly about this; feel free to ignore.

Reply:
Yeah, that's a bit awkward, I agree. I might just get rid of `INT` entirely, and use something like `(SIGNED_INT | UNSIGNED_INT) ~ STAR` as the rule instead. Pest has been helpful, but it's really unwieldy. I'll experiment!

Reply:
Yeah, that looks good :+1: